lms/djangoapps/experiments/utils.py

```diff
@@ -78,6 +98,80 @@ def check_and_get_upgrade_link_and_date(user, enrollment=None, course=None):
 
 
 # TODO: clean up as part of REVEM-199 (START)
+def get_program_purchase_url(skus):
+    """
+    Return a url that will allow the purchase of the courses with these skus
+    """
+    if not skus:
+        return None
+
+    url = 'https://ecommerce.edx.org/basket/add/?'
+    for sku in skus:
+        url += 'sku=' + sku + '&'
+
+    # Remove trailing &
+    return url[:-1]
+
+
+def get_program_price_and_skus(courses):
+    """
+    Get the total program price and purchase skus from these courses in the program
+    """
+    program_price = Decimal('0')
+    skus = []
+
+    for course in courses:
+        course_price, course_sku = get_course_price_and_sku(course)
+        if course_price is not None and course_sku is not None:
+            program_price = Decimal(program_price) + Decimal(course_price)
+            skus.append(course_sku)
+
+    if program_price <= Decimal('0'):
+        program_price = None
+        skus = None
+    else:
+        program_price = unicode(format_course_price(program_price))
```

Comment:
do we need to make sure the arguments to the format string are properly decoded to work with a unicode string? I think this is probably not an issue if the cast to unicode happens after the format but may still be worthwhile to double check